rs/sns/swap/proto/ic_sns_swap/pb/v1/swap.proto

```diff
@@ -275,6 +275,15 @@ message Swap {
 
   // Amount of contributions from the Neurons' Fund committed to this SNS so far.
   optional uint64 neurons_fund_participation_icp_e8s = 20;
+
+  // Information about the timers that perform periodic tasks of this Swap canister.
+  optional Timers timers = 22;
+}
+
+message Timers {
+  optional bool requires_periodic_tasks = 1;
```

Comment:
This field seems redundant. You seem to always copy it from `swap().requires_periodic_tasks()`.

Reply:
The semantics of this field is slightly different, in that the external observer can check whether `swap().requires_periodic_tasks()` has already been read by the scheduled task. In particular, this is used to check that the timers are actually cancelled after the Swap finalization in `test_swap_reset_timers`.